ompi/mca/pml/ob1/pml_ob1_sendreq.h

```diff
@@ -45,11 +45,11 @@ struct mca_pml_ob1_send_request_t {
     mca_pml_base_send_request_t req_send;
     mca_bml_base_endpoint_t* req_endpoint;
     opal_ptr_t req_recv;
-    int32_t req_state;
-    int32_t req_lock;
-    bool req_throttle_sends;
-    size_t req_pipeline_depth;
-    size_t req_bytes_delivered;
+    int32_t  req_state;
+    int32_t  req_lock;
+    bool     req_throttle_sends;
+    int32_t  req_pipeline_depth;
```

Comment:
Why is req_pipeline depth different than the other variables that require ADD_SIZE_T atomics?  It seems like either we have a more general problem (which I could believe) or we don't have a problem that needs fixing in this commit...

Reply:
There is no need to have it as a size_t as it counts the number of descriptors inflight (4 by default).